src/core/thread/csl_tx_scheduler.cpp

```diff
@@ -66,6 +66,18 @@ CslTxScheduler::CslTxScheduler(Instance &aInstance)
     , mFrameContext()
     , mCallbacks(aInstance)
 {
+    InitFrameRequestAhead();
+}
+
+void CslTxScheduler::InitFrameRequestAhead(void)
+{
+    uint32_t busSpeedHz = otPlatRadioGetBusSpeed(&GetInstance());
+    // longest frame on bus is 127 bytes with some metadata, use 150 bytes for bus Tx time estimation
+    uint32_t busTxTimeUs = ((busSpeedHz == 0) ? 0 : (150 * 8 * 1000000 + busSpeedHz - 1) / busSpeedHz);
+
+    // Use ceiling to get next closest integer
+    mCslFrameRequestAhead =
```

Comment:
For non-RCP, set it to a constant for `Mac` handling delay?

Reply:
For non-RCP, the busSpeedHz will be 0 so that it will not add any bus transmission time to it. It will just use the `OPENTHREAD_CONFIG_MAC_CSL_REQUEST_AHEAD_THRESHOLD_US`.

Reply:
OK.